### PR TITLE
Added wholebody and weighting based on local centrality metrics

### PR DESCRIPTION
**Pull request by DuncanZauss**
- COCO wholebody dataset plugin including evaluation with xtpycocotools and pretrained torchhub downloadable shufflenets
- Training weights based on local harmonic centrality for apollocar3d and COCO wholebody

### CODE REVIEW

**Review by svenkreiss — APPROVED** [review]
Excellent. Thanks!
